docs/eventing/brokers/broker-admin-config-options.md

```diff
@@ -1,3 +1,439 @@
+the backing Channel for the Broker class:
+
+    ```yaml
+    apiVersion: eventing.knative.dev/v1
+    kind: Broker
+    metadata:
+      annotations:
+        eventing.knative.dev/broker.class: MTChannelBasedBroker
+      name: default
+      namespace: default
+    spec:
+      config:
+        apiVersion: v1
+        kind: ConfigMap
+        name: config-br-default-channel
+        namespace: knative-eventing
+    ```
+
+### Configuring the default BrokerClass for the cluster
+
+You can configure the `clusterDefault` Broker class so that any Broker created in the cluster that does not have a `BrokerClass` annotation uses this default class.
+
+#### Example
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-br-defaults
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+data:
+  # Configures the default for any Broker that does not specify a spec.config or Broker class.
+  default-br-config: |
+    clusterDefault:
+      brokerClass: MTChannelBasedBroker
+```
+
+### Configuring the default BrokerClass for namespaces
+
+You can modify the default Broker class for one or more namespaces.
+
+For example, if you want to use a `KafkaBroker` class for all other Brokers created on the cluster, but you want to use the `MTChannelBasedBroker` class for Brokers created in `namespace-1` and `namespace-2`, you would use the following ConfigMap settings:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-br-defaults
+  namespace: knative-eventing
+  labels:
+    eventing.knative.dev/release: devel
+data:
+  # Configures the default for any Broker that does not specify a spec.config or Broker class.
+  default-br-config: |
+    clusterDefault:
+      brokerClass: KafkaBroker
+    namespaceDefaults:
+      namespace1:
+        brokerClass: MTChannelBasedBroker
+      namespace2:
+        brokerClass: MTChannelBasedBroker
+```
+
+## Integrating Istio with Knative Brokers
+
+### Protect a Knative Broker by using JSON Web Token (JWT) and Istio
+
+#### Prerequisites
+
+- You have installed Knative Eventing.
+- You have installed Istio.
+
+#### Procedure
+
+1. Label the `knative-eventing` namespace, so that Istio can handle JWT-based user authentication, by running the command:
+
+    ```bash
+    kubectl label namespace knative-eventing istio-injection=enabled
+    ```
+
+1. Restart the broker ingress pod, so that the `istio-proxy` container can be injected as a sidecar, by running the command:
+
+    ```bash
+    kubectl delete pod <broker-ingress-pod-name> -n knative-eventing
+    ```
+
+    Where `<broker-ingress-pod-name>` is the name of your broker ingress pod.
+
+    The pod now has two containers:
+
+    ```bash
+    knative-eventing     <broker-ingress-pod-name>           2/2     Running   1              175m
+    ```
+
+1. Create a broker, then use get the URL of your broker by running the command:
+
+    ```bash
+    kubectl get broker -A
```

Comment:
the `-A` gives all brokers, on the cluster.  (we could also just query the name of the broker being created)

I'd perhaps also state the actual output, e.g.:

```
kubectl get broker -A 
NAMESPACE   NAME        URL                                                                          AGE   READY   REASON
default     my-broker   http://broker-ingress.knative-eventing.svc.cluster.local/default/my-broker   6s    True    
```
since this shows the URL